rllib/tests/test_nested_action_spaces.py

```diff
@@ -79,7 +79,7 @@ def test_nested_action_spaces(self):
                 config["env_config"] = {
                     "action_space": action_space,
                 }
-                for flatten in [False, True]:
+                for flatten in [True, False]:
```

Comment:
Same question here. Does this fix the MARWIL issue or related to the failing nested_action_spaces test?

Reply:
sorry this is no-op, I switched the 2 during testing. I can put them back though if you prefer.
and year, this is only for the failing nested_actions_space test.

Reply:
no, no worries, just wanted to understand it has something to do with the fix :)